Makefile

```diff
@@ -312,7 +312,7 @@ verify-manifests: manifests $(YQ) ## Verify manifests update.
 	yq e -e '.metadata.annotations.containerImage == "$(OPERATOR_IMAGE)"' config/manifests/bases/authorino-operator.clusterserviceversion.yaml
 
 .PHONY: verify-bundle
-verify-bundle: bundle $(YQ) ## Verify bundle update.
+verify-bundle: $(YQ) ## Verify bundle update.
```

Comment:
This virtually makes running `make verify-bundle` locally almost completely useless, but that's OK. In the end, locally, one can always run `make bundle` and thus ensure that the manifests are correct and up to date. Because, otherwise, they have to be committed and included with the changes.

`verify-bundle` only checks if the latest _committed_ bundle manifests need update, by re-running `make bundle` and checking if that leaves any unstaged diff. Perhaps this makes more sense in remote than to simply check locally if you didn't forget running `make bundle` before pushing your changes.

So let's make this target indicated for usage by the automated workflows only and not so much by devs locally.